content/en/docs/instrumentation/java/automatic/extensions.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Extensions
+aliases:
+  - /docs/instrumentation/java/extensions/
```

Comment:
Nit: when there's a single alias, I prefer the compact one-line form (and we can drop the trailing `/`):

```suggestion
aliases: [/docs/instrumentation/java/extensions]
```